### PR TITLE
Calli stuff

### PR DESCRIPTION
**Pull request by Baev1**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,7 @@ module github.com/auttaja/discordgo
 go 1.12
 
 require (
+	github.com/bwmarrin/discordgo v0.19.0
```
Comment (Kelwing) [inline go.mod]: Wait... whats going on here?

Comment (Baev1) [reply go.mod]: good question, idk, I did not put that in afaik, like lol wtf

File: user.go [hunk 2]
```diff
@@ -48,7 +51,7 @@ func (u *User) String() string {
 
 // Mention return a string which mentions the user
 func (u *User) Mention() string {
-	return "<@" + u.ID + ">"
+	return "<@!" + u.ID + ">"
```
Comment (vladfrangu) [inline user.go]: This should be done on the member, and only if they have a nickname...

Comment (Baev1) [reply user.go]: oh, yeah true, will fix

File: go.sum [hunk 3]
```diff
@@ -1,3 +1,5 @@
+github.com/bwmarrin/discordgo v0.19.0 h1:kMED/DB0NR1QhRcalb85w0Cu3Ep2OrGAqZH1R5awQiY=
```
Comment (Kelwing) [inline go.sum]: Whats up with this??

Comment (Kelwing) [inline go.sum]: Whats up with this?

**Pull request merged**